src/components/ReviewConfirm/Certify.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <v-card flat id="step-5-container">
+    <div class="certify-container">
+      <div class="certifiedby-container">
+        <label>Legal Name</label>
+        <div class="value certifiedby">
+          <v-text-field
+            filled
+            persistent-hint
+            id="certified-by-textfield"
+            label="[Default Completing Party value]"
+            hint="Enter the Name of the Completing Party"
+            :value="certifiedBy"
+            @input="emitCertifiedBy"
+          />
+        </div>
+      </div>
+      <v-checkbox
+        :value="isCertified"
+        @change="emitIsCertified"
+      >
+        <template slot="label">
+          <div class="certify-stmt">
+            I, <b>{{trimmedCertifiedBy || '[Legal Name]'}}</b>, {{certifyStatementResource.certifyStatementHeader}}
+          </div>
+        </template>
+      </v-checkbox>
+      <p>
+        <ul class="certify-statements">
+          <li v-for="(statement, index) in certifyStatementResource.certifyStatements" v-bind:key="index">
+              {{ statement }}
+          </li>
+        </ul>
+      </p>
+      <p></p>
+      <p class="certify-date">Date: {{date}}</p>
+      <p class="certify-clause">
+        {{certifyStatementResource.certifyClause}}
+      </p>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import { State } from 'vuex-class'
+import { CertifyStatementIF } from '../../interfaces'
```

Comment:
I believe we can reference the import `@/interfaces`

Reply:
Sure. I think these were auto imported. Will change the relative path